src/_includes/article.html

```diff
@@ -61,8 +61,9 @@ <h1 class="post_heading">
                 &ndash;
             </span>
         {% endif %}
-            <span class="post_date date">
-                {{ post.date|date("%b %d, %Y") }}
+            <span class="date">
+                {% import "macros/time.html" as time %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5cdae78991a4cb487fbe2288b7fbb10132e9a955 Let me know if I missed any.